docker-hub/api/latest.yaml

```
@@ -32,6 +32,10 @@ tags:
     description: |
       The Advanced Image Management API endpoints allow you to manage Docker images across all repositories.
       For more information, see [Advanced Image Management dashboard](https://docs.docker.com/docker-hub/image-management/).
+  - name: Audit Logs
+    x-displayName: Audit Logs
+    description: |
+      The Audit Logs API endpoints allow you to query audit log events across your repositories and organizations.
```

```suggestion
      The Audit Logs API endpoints allow you to query audit log events across your repositories and organizations.
      For more information, see [Audit log](https://docs.docker.com/docker-hub/audit-log/).
```
Add link to Hub UI doc